db/migrate/20230523170858_create_like.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Migration to create the likes table
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true, index: true
```

Comment:
- [ ] you should reference to author as well similar to what you did in the Post table:
```
t.references :author, null: false, foreign_key: { to_table: :users }, index: true
```